examples_utils/benchmarks/environment_utils.py

```diff
@@ -9,6 +9,8 @@
 # Get the module logger
 logger = logging.getLogger(__name__)
 
+POPRUN_VARS = ["HOSTS", "PARTITION", "CLUSTER", "TCP_IF_INCLUDE", "VIPU_CLI_API_HOST"]
```

Comment:
```suggestion
POPRUN_VARS = ["HOSTS", "PARTITION", "CLUSTER", "TCP_IF_INCLUDE", "VIPU_CLI_API_HOST", "IPUOF_VIPU_API_HOST", "IPUOF_VIPU_API_PARTITION_ID"]
```

I currently have a diff to change the poprun commands to replace the `PARTITION` and `VIPU_CLI_API_HOST` variables to the standard IPUOF ones expected by poplar

Reply:
I'd refactor this and tie the environment variables to their help strings, probably with a dictionary:

```python
POPRUN_VARS = {
    "HOSTS": "Comma seperated list of IP addresses/names of "
                  "the machines you want to run on. Try to copy across "
                  "ssh-keys before attempting if possible. e.g. "
                  "10.1.3.101,10.1.3.102,... or lr17-1,lr17-2,...\n",
...
}
```

and then use that to build the error message dynamically?

Reply:
I like this idea, thanks, will implement